packages/astro/src/@types/astro.ts

```diff
@@ -822,6 +822,39 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+
+		/**
+		 * @docs
+		 * @name build.split
+		 * @type {boolean}
+		 * @default {false}
+		 * @version 2.7.0
+		 * @description
+		 * Defines how the SSR code should be bundled.
+		 *
+		 * When `split` is `true`, Astro will emit a file for each page.
+		 * Each file emitted will render only one page. The pages will be emitted
+		 * inside a `dist/pages/` directory, and the emitted files will keep the same file paths
+		 * of the `src/pages` directory.
+		 *
+		 * Inside the `dist/` directory, the pages will look like this:
+		 * ```plaintext
+		 * ├── pages
+		 * │   ├── blog
+		 * │   │   ├── entry._slug_.astro.mjs
+		 * │   │   └── entry.about.astro.mjs
+		 * │   └── entry.index.astro.mjs
+		 * ```
```

Comment:
I'm going to ask @delucis to weigh in on this one:

Chris: will this plaintext file tree be picked up and rendered with our `<Filetree />` component in docs? If not, this will look out of place by comparison.

I might consider just not showing it at all, since we do show this nice component file tree in docs itself on the SSR page. Maybe it's good enough for reference to just say that `dist/pages/` mirrors `src/pages/`?

Reply:
Ok, so just to clarify, this will NOT show up in the `@description`? If not, then do whatever you want as it won't get published in docs! :smile:

Reply:
Sorry that I was a wish of mine :D 
**Yes, it will be shown in the `@description`**

Reply:
@sarah11918 I removed the `plaintext` block from the documentation